moveit_core/collision_distance_field/include/moveit/collision_distance_field/collision_distance_field_types.h

```diff
@@ -69,6 +69,7 @@ enum CollisionType
 
 struct CollisionSphere
 {
+  using AlignedVector = std::vector<CollisionSphere, Eigen::aligned_allocator<CollisionSphere>>;
```

Comment:
I suggest renaming `AlignedVector` to just `Vector`. Throughout the API we are not interested in the reason for this data type anymore. Here, however, a comment will be helpful.